airbyte-scheduler/client/src/main/java/io/airbyte/scheduler/client/SchedulerJobClient.java

```diff
@@ -27,7 +27,7 @@
 import io.airbyte.config.DestinationConnection;
 import io.airbyte.config.SourceConnection;
 import io.airbyte.config.StandardSync;
-import io.airbyte.scheduler.Job;
+import io.airbyte.scheduler.data.Job;
```

Comment:
I can't see this file, so I assume you didn't add it to the PR yet.

Same comment as for persistence. why not put it in the `models` module?

Also quick question, would it make sense to keep models & persistence in the same jar?

Reply:
i'll double check on this. probably just a mistake when i moved files.

I don't think `models` and `persistence` should be in the same jar. Really the `persistence` package should not exist at all (or if it did exist it would only be depended on by the `main` package). It is currently depended on by both `main` and `client` because the `client` can only communicate with `main` through the database. It is really an anti pattern imo that the client depends on the database at all. But I am not going to fix that problem in this PR.

If you think about the module setup in a microservice environment we might do something like this.
* models (anything in the world should be allowed to depend on this)
* client (depends on models) (anything outside of this module can depend on this)
* persistence (i.e. the database client) (should only be depended on internally)
* server (depends on models, persistence) (nothing should depend on this)
* worker / main (depends on models, persistence) (nothing should depend on this)

because we don't have a server module, client has to depend on persistence 😭 , but we should strive to change this, in which case it will make sense for models and persistence to be separate. we really don't want external things interacting with the scheduler database.

Reply:
Spoke offline. Leaving a comment here for tracking.

Same thoughts as Michel for me - models and persistence should be in the same JAR instead of separate packages.

Reply:
the goal is that the persistence will eventually not be depended upon by the client or any external module. for now we deal with it.